odoo/addons/base/tests/test_orm.py

```diff
@@ -97,6 +98,15 @@ def test_access_filtered_records(self):
         with self.assertRaises(AccessError):
             (p1 + p2).with_user(user).unlink()
 
+    def test_call_kw(self):
+        partner = self.env['res.partner'].create({'name': 'MyPartner1'})
+        args = (partner.ids, ['name'])
+        kwargs = {'context': {'test': True}}
+        kwargs_copy = kwargs.copy()
+        res = api.call_kw(self.env['res.partner'], 'read', args, kwargs)
+        self.assertEqual(res, partner.read(['name']))
+        self.assertEqual(kwargs, kwargs_copy)
```

Comment:
Could be simplified as
```suggestion
        kwargs = {'context': {'test': True}}
        api.call_kw(self.env['res.partner'], 'read', args, kwargs)
        self.assertEqual(kwargs, {'context': {'test': True}})
```